src/index.js

```diff
@@ -0,0 +1,213 @@
+
+
+const currentState = {
+    city: "Herndon",
+    latitude: 38.9696,
+    longitude: 77.3861,
+    temp: 50,
+    weather: "sunny"
+}
+
+
+////////// API calls //////////
+const getWeatherFromCity = () => {
+    axios
+        .get(`http://localhost:5000/location`, {
+            params: {
+                q: currentState.city
+            },
+        })
+        .then((response) => {
+            currentState.latitude = response.data[0]['lat'];
+            currentState.longitude = response.data[0]['lon'];
+            console.log(`getting weather for ${currentState.city}`)
+            // call the other api
+            getWeatherFromCoords();
+
+        })
+        .catch((error) => {
+            console.log(`error getting coords from ${currentState.city}`)
+            console.log(error)
+        });
+    }
+
+
+    const getWeatherFromCoords = () => {
+        axios 
+            .get(`http://localhost:5000/weather`, {
+                params: {
+                    lat: currentState.latitude,
+                    lon: currentState.longitude
+                }
+            })
+            .then((response) => {
+                // get temp and update
+                kelvinTemp = response.data["main"]["temp"]
+                tempDegF = kelvinToDegF(kelvinTemp)
+                console.log(`finding temp from ${currentState.city} as ${tempDegF}`)
+                updateTemp(tempDegF);
+    
+                // get weather and update
+                weather = response.data["weather"][0]["description"];
+                currentState.weather = weather;
+                updateSky(weather);
```

Comment:
Nice solution here. How might you refactor this so the `updateTemp` logic and `updateSky` logic are nested in one function so our code can be more dry.